src/app/components/mnoe-api/admin/organizations.svc.coffee

```diff
@@ -20,6 +20,16 @@
     _.forEach observerCallbacks, (callback) ->
       callback(listPromise)
 
+  @organizations = (limit, offset, sort, params = {}) ->
+    return _getOrganizations(limit, offset, sort, params)
```

Comment:
`_getOrganizations()` isn't used anywhere else. You can move its content directly here.